### ISSUE TITLE
Issue with gps

### ISSUE THREAD

**Issue by Annika-wyt** [KTH-SML/svea#38]
Unable to get updated rtk gps.
The ntrip_client updated the subscriber and publisher topic from /ntrip_client/nmea to /nmea and from /ntrip_client/rtcm to /rtcm.

**Comment by kaarmu** [comment]
So this was fixed today?

**Comment by Annika-wyt** [comment]
Yes, just put a remark here before I am ready to make a pull request for the update.
